README.md

```diff
@@ -1,8 +1,67 @@
-# RegressionFinder
+# [Intern Project] RegressionFinder
+
+An automatic tool which finds regressions in Chrome browser.
+
+## Table                                                                                     
+1. [Setup](#Setup)    
+2. [Tutorial](#Tutorial)                                                                                                       
+
+## Setup
+- You need to install the dependencies of RegressionFinder.
+- The bash file `setup.sh` will install all of dependencies.
+
+```shell
+$ git clone https://github.com/SuhwanSong/RegressionFinder.git
+$ cd RegressionFinzer
+$ ./setup.sh
+```
+
+## Tutorial
+- First, you need to generate the html files.
+
+```shell
+$ cd RegressionFinzer/src/domato
+$ python3 generator.py --output_dir ./html_testcases --no_of_files 10000
+```
+* This will generate 10,000 testcases in `./html_testcases` directory. 
+* Now you can run RegressionFinder with `r2z2.py`.
+* It provides six options:
```

Comment:
Is it possible to add base and target flags so the following works?
```-b 106 --base-flags="" -t 106 --target-flags="--enable-blink-features=LayoutNG"```
If yes, can you add support for this in a separate change?

Reply:
It's possible to add that support.
Do you think the value of option -b should be the beginning revision of Chrome version and the value of option -t should be the latest revision of Chrome version?
(e.g., if -b: 80, -t: 80 -> revision range: [706893, 722269])

Or like -b: 80, -t: 80 -> revision range: [706893, 706893])



--80-----------81---

Reply:
And when only target browser uses additional flag, how to bisect that?